packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -73,13 +69,12 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     this._lastValidValue = value;
 
     // Ensure that the autocalculated precision is not negative.
-    const precision = props.precision || Math.max(calculatePrecision(props.step!), 0);
+    this._precision = props.precision || Math.max(calculatePrecision(props.step!), 0);
```

Comment:
Ideally the precision calculation should be done in a helper function so it's not duplicated (and slightly different) between the constructor and componentWillReceiveProps. Also, the helper ought to do a check for precision being of type number (not just a truthy/falsey check) so it correctly detects a specified precision of 0.